specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,316 @@
+# Background
+
+We are exposing an event that will be raised when an attempt to launch a protocol that is registered with the OS (external protocol) is made. The host will be given the option to cancel the launch. Cancelling the launch gives the host the opportunity to hide the default dialog, display a custom dialog, and then launch the external protocol themselves.

```

Comment:
Please break lines longer than 80 or 100 characters. It makes it harder for reviewers to read